.circleci/config.yml

```diff
@@ -278,107 +305,6 @@ commands:
-            ng test toolkit --watch=false --browsers ChromeHeadlessNoSandbox --reporters junit
-            ng test $ANGULAR_PROJECT_NAME --watch=false --browsers ChromeHeadlessNoSandbox --reporters junit
-          environment:
-            JUNIT_REPORT_PATH: /tmp/junit/
-            JUNIT_REPORT_NAME: test-results.xml
-          when: always
-      - store_test_results:
-          path: /tmp/junit
-      - store_artifacts:
-          path: /tmp/junit
-
-  build-atcp-assets:
-    description: build atcp auth0 scripts & styles
-    steps:
-      - run:
-          name: build atcp auth0 scripts
-          command: |
-            set -u
-            ng build ddp-atcp-auth
-            npm run convert-atcp-auth-build-to-UMD
-      - run:
-          name: build atcp auth0 styles
-          command: |
-            set -u
-            ./node_modules/.bin/sass ./projects/ddp-atcp/src/auth0/auth0-styles/main.scss ./projects/ddp-atcp/src/auth0/compiled/auth0-styles/main.css --style compressed
-
-  execute-study-specific-commands:
-    description: run study specific commands if necessary
-    parameters:
-      study_key:
-        type: string
-        default: "UNKNOWN"
-    steps:
-      - when:
-          condition:
-            equal: [ atcp, << parameters.study_key >> ]
-          steps:
-            - build-atcp-assets
-
-  build-app:
-    description: "Build DDP study app"
-    parameters:
-      study_key:
-        type: string
-        default: "UNKNOWN"
-    steps:
-      - when:
-          condition:
-            or:
-              - equal: [ atcp, << parameters.study_key >> ]
-          steps:
-            - execute-study-specific-commands:
-                study_key: << parameters.study_key >>
-      - run:
-          name: ng build
-          command: |
-            set -u
-            export NODE_OPTIONS="--max-old-space-size=4096"
-            ng build $ANGULAR_PROJECT_NAME $BUILD_PARAMS --prod --aot --base-href=/ --output-path=$OUTPUT_DIR --verbose=true --progress=true
-
-  store-app-build:
-    description: "Copy build to cloud storage"
-    steps:
-      - run:
-          name: Create build archive
-          command: |
-            set -u
-            DATE=`date +%F`
-            TAR_NAME="${ANGULAR_PROJECT_NAME}_${DATE}_${SHORT_GIT_SHA}.tar.gz"
-            TAR_PATH="/tmp/${TAR_NAME}"
-            # Save some ENV vars for in job downstream
-            echo "export TAR_NAME=$TAR_NAME" >> $BASH_ENV
-            echo "export TAR_PATH=$TAR_PATH" >> $BASH_ENV
-            cd ${OUTPUT_DIR}
-            tar -czvf ${TAR_PATH} .
-            echo "Tar file created at ${TAR_PATH}"
-            ls -l $TAR_PATH
-      - run:
-          name: Store build archive
-          command: |
-            set -u
-            readvault.sh secret/pepper/prod/v1/conf .data.gcp.serviceKey | gcloud auth activate-service-account --key-file=-
-            gsutil cp  ${TAR_PATH} "gs://${BUILDS_BUCKET}/${ANGULAR_PROJECT_NAME}/${TAR_NAME}"
```

Comment:
`store-app-build` command is used only by `app-build-and-store-job` job. Steps are combined into the job.